### PR TITLE
Shift from npm to yarn

### PR DESCRIPTION
**Pull request by kanufy**
Fix: #89

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,29 @@
 sudo: false
 
-language: node_js
-
-install:
-  - npm install
+notifications:
+  email: false
```
Comment (nekoya) [inline .travis.yml]: この`notifications`と今回の範囲から外れるけど`node_js`の項目が、他とインデントが違うっぽい。

ついでなので調整しちゃいましょうか。

Comment (kanufy) [inline .travis.yml]: お、ほんとですね。
なおしますわ

Comment (kanufy) [inline .travis.yml]: :done:

**Comment by nekoya** [comment]
@popuko r+

**Comment by popuko** [comment]
:pushpin: Commit 5eb53e4b290bb3d0635914584417df37c5a23c93 has been approved by `nekoya`

**Comment by popuko** [comment]
:hourglass: 5eb53e4b290bb3d0635914584417df37c5a23c93 has been merged into the auto branch 924badf7cabe5725173079f47bb1b6e3e68881ed

**Comment by popuko** [comment]
:tada: The result of what tried to merge this pull request is `success`.

* [The Travis CI build passed](https://travis-ci.org/voyagegroup/eslint-config-fluct/builds/244419104?utm_source=github_status&utm_medium=notification)

**Pull request merged**